test/functional/MyClassFuncTest.php

```diff
@@ -3,29 +3,28 @@
 namespace {{ns}}\FuncTest;
 
 use Xpmock\TestCase;
+use {{ns}}\MyClass as TestSubject;
 
 /**
  * Tests {@see {{ns}}\MyClass}.
  *
- * @TODO Search and replace tokens that begin with underscore.
- *
  * @since [*next-version*]
  */
-class MyClassTest extends TestCase
+class MyClassFuncTest extends TestCase
```

Comment:
Unnecessary, as it's already in the `FuncTest` namespace. Also, does not follow the general pattern; see `use` above.